col/src/main/java/vct/col/resolve/lang/Java.scala

```diff
@@ -453,4 +469,79 @@ case object Java extends LazyLogging {
 
   def double[G](implicit o: Origin = DiagnosticOrigin): TFloat[G] = TFloats.ieee754_64bit
   def float[G](implicit o: Origin = DiagnosticOrigin): TFloat[G] = TFloats.ieee754_32bit
-}
\ No newline at end of file
+}
+
+sealed trait JavaAnnotationData[G]
```

Comment:
Not sure where to place this in the folder hierarchy so it wound up in Java.scala